src/main/java/com/example/books/dto/cartitem/CartItemDto.java

```diff
@@ -2,12 +2,18 @@
 
 import jakarta.validation.constraints.Min;
 import jakarta.validation.constraints.NotNull;
+import lombok.Data;
 
-public record CartItemDto(
-        @NotNull
-        Long bookId,
-        @NotNull
-        @Min(1)
-        int quantity
-) {
+@Data
+public class CartItemDto {

```

Comment:
I don't like approach where part of DTOs are records and part are simple classes.
This is inconsistent approach and each time you use new dto class you should check how to work with it.
Please make all of DTOs either records or classes